LandSandBoat/server
# [Item Mods] Add Double/Triple Attack Damage + mods and functionality

[Item Mods] Add Double/Triple Attack Damage + mods and functionality



**_I affirm:_**

## What does this pull request do?


- Creates new Double Attack and Triple Attack mods.
- Adjusts double/triple attack core functionality to ascribe all hits of a double/triple attack the PHYSICAL_ATTACK_TYPE::DOUBLE or PHYSICAL_ATTACK_TYPE::TRIPLE attributes. This allows the core to apply the above mods to all hits of the double/triple attack.

## PR Description


- Addresses the addition and handling of two new item mods.

## Steps to test these changes

!setmod 286 1-100 charName (Sets DOUBLE_ATTACK_DMG to specified value)
!setmod 287 1-100 charName (Sets TRIPLE_ATTACK_DMG to specified value)
!setmod 288 1-100 charName (sets DOUBLE_ATTACK proc rate to specified value)
!setmod 302 1-100 charName (sets TRIPLE_ATTACK proc rate to specified value)

Go to the mob/NM of your choice, set your mods, and start whacking away. Changing the value of the DOUBLE_ATTACK_DMG or TRIPLE_ATTACK_DMG should increase the damage of all double/triple attack white damage hits.



## References
Double Attack Damage - https://www.bg-wiki.com/ffxi/War._Cuisses_%2B2
Double Attack Damage - https://www.bg-wiki.com/ffxi/Agoge_Cuisses_%2B3
Triple Attack Damage - https://www.bg-wiki.com/ffxi/Asn._Poulaines_%2B2
